pom.xml

```diff
@@ -517,7 +517,7 @@
 			<dependency>
 				<groupId>net.sf.ehcache</groupId>
 				<artifactId>ehcache</artifactId>
-				<version>2.10.10.1.14</version>
+				<version>2.10.9.2</version>
```

Comment:
Any reason of  downgrading ehcache version ?

Reply:
when I made git pull --rebase upstream master after creating the branch  dependabot  had added  2.10.10.1.14 on remote master which wasn't working on the ehcache imports (even 2.10.10.1.14 v seems not yet out . link https://mvnrepository.com/artifact/net.sf.ehcache/ehcache)